src/pint/models/timing_model.py

```diff
@@ -406,12 +414,21 @@ def free_params(self, params):
 
     def match_param_aliases(self, alias):
         """Return the parameter corresponding to this alias."""
-        for p in self.params:
+        # Search the top level first.
+        for p in self.top_level_params:
             if p == alias:
                 return p
             if alias in getattr(self, p).aliases:
                 return p
-        raise ValueError("{} is not recognized as a parameter or alias".format(alias))
```

Comment:
Raising an exception is much more pythonic and avoids the headaches caused by returning None. If you want to make it easier to catch you can create a custom subclass of ValueError.